README.md

```diff
@@ -107,6 +107,7 @@ const initAuth = () => {
       authDomain: 'my-example-app.firebaseapp.com',
       databaseURL: 'https://my-example-app.firebaseio.com',
       projectId: 'my-example-app-id',
+      tenantId: 'example-tenant-id', // Optional, only necessery in multi-tenant configuration
```

Comment:
Change to commented-out by default because many users copy-paste the config. Fix typo in "necessary".
`// tenantId: 'example-tenant-id', // Optional, only necessary in multi-tenant configuration`